### PR TITLE
E2E: improve P00002 for spiderpool

### PR DESCRIPTION
**Pull request by ty-dc**
Signed-off-by: ty-dc <tao.yang@daocloud.io>

What type of PR is this?
/kind  pr/release/none-required

What this PR does / why we need it:

- An enhancement to P00002 specifying the creation of a certain number of ippools to verify their impact on IP allocation.

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:
- Number of copies and number of ippools customisable

Additional documentation e.g.,  development documentation, usage docs, etc.:
```
None
```

### CODE REVIEW

File: test/e2e/performance/performance_test.go [hunk 1]
```diff
@@ -41,138 +45,135 @@ var _ = Describe("performance test case", Serial, Label("performance"), func() {
...
 				AddReportEntry("Performance Results",
-					fmt.Sprintf(`{ "controllerType" : "%s", "replicas": %d, "createTime": %d , "rebuildTime": %d, "deleteTime": %d }`,
-						controllerType, replicas, int(endT1.Seconds()), int(endT2.Seconds()), int(endT3.Seconds())))
+					fmt.Sprintf(`{ "controllerType" : "%s", "replicas": %d, "iPPoolNumber": %d , "createTime": %d , "rebuildTime": %d, "deleteTime": %d }`,
+						controllerType, replicasOrIpNum, iPPoolNum, int(endT1.Seconds()), int(endT2.Seconds()), int(endT3.Seconds())))
 			},
-			// TODO (tao.yang), N controller replicas in Ippool for N IP, Through this template complete gc performance closed-loop test together
-			Entry("time cost for creating, rebuilding, deleting deployment pod in batches",
-				Label("P00002"), common.DeploymentNameString, int32(60), time.Minute*4),
+			Entry("Time cost of creating, rebuilding and deleting deployment pods in batches",
```
Comment (weizhoublue) [inline test/e2e/performance/performance_test.go:175]: 要有 ippool =0  作为 性能 比对

Comment (ty-dc) [reply test/e2e/performance/performance_test.go:175]: 是想过的，这样加了，但整个ci的时间会增加非常长吧？
ipv4、IPv6、双栈

File: test/e2e/performance/performance_test.go [hunk 2]
```diff
@@ -41,138 +45,135 @@ var _ = Describe("performance test case", Serial, Label("performance"), func() {
...
-				GinkgoWriter.Printf("time cost for create  %v: %v/%v of %v replicas = %v \n", controllerType, nsName, perName, replicas, endT1)
-				GinkgoWriter.Printf("time cost for rebuild %v: %v/%v of %v replicas = %v \n", controllerType, nsName, perName, replicas, endT2)
-				GinkgoWriter.Printf("time cost for delete  %v: %v/%v of %v replicas = %v \n", controllerType, nsName, perName, replicas, endT3)
-				// attaching Data to Reports
+				// Output the performance results
+				GinkgoWriter.Printf("When the cluster has %v ippools, the time cost of creating %v replicas of the %v:%v/%v is %v \n", iPPoolNum, replicasOrIpNum, controllerType, nsName, perName, endT1)
+				GinkgoWriter.Printf("When the cluster has %v ippools, the time cost of rebuilding %v replicas of the %v:%v/%v is %v \n", iPPoolNum, replicasOrIpNum, controllerType, nsName, perName, endT2)
+				GinkgoWriter.Printf("When the cluster has %v ippools, the time cost of deleting %v replicas of the %v:%v/%v is %v \n", iPPoolNum, replicasOrIpNum, controllerType, nsName, perName, endT3)
+				// Attaching Data to Reports
 				AddReportEntry("Performance Results",
```
Comment (weizhoublue) [inline test/e2e/performance/performance_test.go:171]: AddReportEntry 需要工具不同的  entry ，有不同的 描述，不然 怎么找报告

**Review by bzsuni — APPROVED** [review]

File: test/e2e/performance/performance_test.go [hunk 3]
```diff
@@ -41,138 +45,141 @@ var _ = Describe("performance test case", Serial, Label("performance"), func() {
...
 			},
-			// TODO (tao.yang), N controller replicas in Ippool for N IP, Through this template complete gc performance closed-loop test together
-			Entry("time cost for creating, rebuilding, deleting deployment pod in batches",
-				Label("P00002"), common.DeploymentNameString, int32(60), time.Minute*4),
+
+			// When the IP and Pod are 1:1, I want all Pods to run successfully.
+			// But I shouldn't change the global default ippool, so I need to generate an ippool.
+			Entry("Time cost of creating, rebuilding and deleting deployment pods in batches when the cluster has only one ippools",
+				Label("P00002"), "Performance Results", common.DeploymentNameString, 60, 1, time.Minute*4),
+			Entry("Time cost of creating, rebuilding and deleting deployment pods in batches when clusters have many ippools",
```
Comment (weizhoublue) [inline test/e2e/performance/performance_test.go:181]: 少加一点吧，再加了这个，一组 E2E 奔着 1个小时去了

**Pull request closed without merging**